app/controllers/api/v1/events/nearests_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::Events::NearestsController < ApiController
+  def index
+    @events = Event.near(
+      [params[:lat], params[:lon]],
+      params[:radius],
+      units: :km
+    )
+    if @events.count(:all) > 0
```

Comment:
Have you tried `Events.exists?`?

```
Loading development environment (Rails 4.1.4)
[1] pry(main)> Referral.exists?
  Referral Exists (0.4ms)  SELECT  1 AS one FROM "referrals"  LIMIT 1
=> true
[2] pry(main)> Referral.any?
   (0.4ms)  SELECT COUNT(*) FROM "referrals"
=> true
```

The `Referral.exists?` does a limit so doesn't have to count all of them, just checks if any records exist in the table.